classes/omnect_uboot_embedded_version.bbclass

```diff
@@ -58,6 +58,10 @@
 #    up to defined size of resulting file; default value is 0x00
 #    however, for NOR flashes it would be better to have an 0xff, therefore
 #    this variable
+#  - OMNECT_BOOTLOADER_EMBEDDED_VERSION_UUUTAG
+#    if bootloader is used for production programming, i.e. uuu tool flashes
+#    boot medium via ROM serial boot, it needs an end tag for the tool to
+#    detect its size.
```

Comment:
rest of this PR makes me believe this is a boolean? if so please document, otherwise an example would be nice to understand a possible value.

Reply:
improved description.